packages/icon/addon/components/nucleus-icon.js

```diff
@@ -59,11 +73,12 @@ class NucleusIcon extends Component {
   * @computed _disabled
   * @private
   */
-  @computed('size', 'customClass')
+  @computed('size', 'variant', 'customClass')
   get _classNames() {
     let customClass = this.get('customClass');
-    let size = this.get('size') ? this.get('size') : 'medium';
-    let classNames = `nucleus-icon nucleus-icon--${size}`;
+    let size = this.get('size') ? this.get('size') : DEFAULT_SIZE;
```

Comment:
Do we need the conditional statement anymore?

Reply:
You're right, we don't need it. Refactored.